modules/gdnative/gdnative_api.json

```diff
@@ -538,6 +538,14 @@
           ["const godot_vector2 *", "p_self"]
         ]
       },
+      {
+        "name": "godot_vector2_direction_to",
```

Comment:
This breaks binary compatibility with all previously build GDNative libraries.

If desired I can start a new GDNative core-extension for this, but in GDNative code it's not as easy as just plugging in code, there's a bit more bookkeeping involved.

If the PR itself is accepted, I can help you set up the extension properly!

Reply:
Hmm.. should I change it somehow now? I did it cuz @reduz is requested to do it. Overall it is a correct procedure to add something to GDNative ?

Reply:
In theory this workflow is correct if the change is for an "unpublished" extension.

GDNative aims to always be binary compatible, so all functionality is passed via structs which contain function pointers.

Once an extension is released the struct *must not* change!

To add new functionality a new extension has to be created, which will be available as a `gdnative_api_struct_extension *next` field (or something like that).

So the correct way is to add a new "extension" to the core API in the json file, if I coded the code generation properly then it should just work.

Reply:
Maybe I remove it for now and you are add it later when you want, ok ? Im not understand how to do it correctly anyways..

Reply:
Yes, seems fine

Reply:
It would be good to have dev documentation on how to expose new APIs to GDNative, as it's a recurring topic.